scripts/installers/make-macpkg

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python
+"""Script to create a mac pkg installer.
+
+This script assumes that an executable has been produced previously
+by the sibling script make-pyinstaller.
+"""
+import argparse
+import os
+import subprocess
+import sys
+
+
+ROOT = os.path.dirname(os.path.dirname(os.path.dirname(
+    os.path.abspath(__file__))))
+PKG_DIR = os.path.join(ROOT, 'macpkg')
+EXE_NAME = 'aws'
+TEMP_PKG_NAME = 'aws-cli.pkg'
+PKG_NAME = 'AWS-CLI-Installer.pkg'
+
+
+class BadRCError(Exception):
+    pass
+
+
+def run(cmd, cwd=None):
+    sys.stdout.write("Running cmd: %s\n" % cmd)
+    kwargs = {
+        'shell': True,
+        'stdout': subprocess.PIPE,
+        'stderr': subprocess.PIPE,
+    }
+    if cwd is not None:
+        kwargs['cwd'] = cwd
+    p = subprocess.Popen(cmd, **kwargs)
+    stdout, stderr = p.communicate()
+    rc = p.wait()
```

Comment:
You don't need `.wait()` if you call `.communicate()`.  You can just access `p.returncode` once `communicate()` returns.